make.js

```diff
@@ -114,7 +124,10 @@ var MOCHA_OPTS = ' --timeout 15000 --slow 500';
                     var res = exec(BOOTLINT + ' -d W013 ' + output);
 
                     echo('+ node make stop');
-                    target.stop();
+                    try {
+                        // it's okay if this fails
+                        target.stop();
+                    } catch (e) { }
```

Comment:
I know this feels hacky, and it kind of is, but honestly it's really doesn't matter if this pass or fails, it has no impact on the bootlint action.